test/es-module/test-esm-loader-chaining.mjs

```diff
@@ -253,6 +253,23 @@ describe('ESM: loader chaining', { concurrency: true }, () => {
     assert.strictEqual(code, 0);
   });
 
+  it('should allow loaders to influence subsequent loader resolutions', async () => {
+    const { code, stderr } = await spawnPromisified(
+      execPath,
+      [
+        '--loader',
+        fixtures.fileURL('es-module-loaders', 'loader-resolve-strip-xxx.mjs'),
+        '--loader',
+        `xxx/${relative(process.cwd(), fixtures.path('es-module-loaders', 'loader-resolve-strip-yyy.mjs'))}`,
```

Comment:
Is it necessary to wrap this in `path.relative()`? (I think no, but if yes, I think you could simply it with [`path.resolve()`](https://nodejs.org/api/path.html#pathresolvepaths), which will form an absolute URL based on `process.cwd()`). Also if yes, perhaps a comment explaining why this is special.

Reply:
Notice the `xxx/` - this test accesses the loader through a bare identifier, not a relative (nor absolute) path.

Reply:
I reworked the code and it should be clearer now 👍